### ISSUE TITLE
Link Tag Intro

### ISSUE THREAD

**Issue by Vertacx** [Vertacx/CO-FEB-0124-Shubasri-Frontend-Development-Upskilling#24]
Link tags in HTML are used to define the relationships between the current document and external resources. 
They are placed within the `<head> `section of an HTML document and can be used for a variety of purposes, including linking to stylesheets, favicons, and providing metadata for search engines and social media.

Video Link : 
[Link Canonical Intro](https://youtu.be/gVE2I0YYdTE)

Basic Syntax of a Link Tag
`<link rel="relationship" href="URL" type="MIME-type">`

**Understanding rel, href, and type**

1. **rel (Relationship)**

    **What it is:** Think of rel as describing the kind of connection or relationship between your current webpage and the linked resource.
    **Example:** If you're linking to a stylesheet (a file that makes your website look nice with colors and fonts), you use rel="stylesheet" to tell the browser, "Hey, this link is to a stylesheet."
    **Analogy:** Imagine you’re sending an invitation. rel is like writing on the envelope whether it's a birthday invite, a wedding invite, or a thank-you card. It tells the recipient what kind of message is inside.
2. **href (Hyperlink Reference)**
**What it is:** href stands for Hyperlink Reference. It's the address where the resource is located, kind of like a street address or a URL.
Example: If you're linking to a file called styles.css, you put the path to that file in href, so the browser knows where to find it.
**Analogy:** Think of href as the destination address you put on a letter. It tells the post office (or in this case, the web browser) where to deliver or find something.
3. **type (Type)**
**What it is:** type specifies the kind of content you are linking to. This helps the browser understand what to do with the resource once it gets there.
**Example:** If you're linking to a CSS file, you might use type="text/css" to tell the browser that the file is a CSS stylesheet.
**Analogy:** Imagine you’re labeling a container in your pantry. Writing "sugar" on a jar tells you what's inside and how you can use it. Similarly, type tells the browser what kind of file it's dealing with and how to use it.

**Bringing It All Together**

Let’s use an example that combines rel, href, and type to show how they work together:
`<link rel="stylesheet" href="styles.css" type="text/css">`

- rel="stylesheet": This part tells the browser, "This link is related to a stylesheet."

- href="styles.css": This part tells the browser, "You can find this stylesheet at the address styles.css."

- type="text/css": This part tells the browser, "The file you will find is a CSS text file."

**Conclusion**
In summary, rel, href, and type in a link tag work together to tell the browser about the kind of link, where to find the resource, and what type of resource it is. These elements help ensure that your webpage can pull in and properly use external resources like stylesheets, scripts, or other linked content.
